src/services/babel-browser.gen.js

```diff
@@ -0,0 +1,78 @@
+const babel = require('@babel/core');
+const presetEnv = require('@babel/preset-env');
+
+const browserList = require('../../config/browsers.json');
+
+module.exports = () => {
+  const browserListForBabel = {};
+
+  for (const browser in browserList) {
+    if (browser !== 'chromium') {
+      browserListForBabel[browser] = browserList[browser];
+    }
+  }
+
+  // preset-env doesn't give an introspection API, so we have to get it from
+  // the metadata on transformed code
+  const result = babel.transformSync('', {
+    sourceType: 'script',
+    presets: [
+      [
+        presetEnv,
+        {
+          targets: browserListForBabel,
+          // The module for regenerator causes an import error, so we ignore it
```

Comment:
Ibid. I think we could self-document by defining a constant like `ERRORING_TARGETS_THAT_ARE_SAFE_TO_EXCLUDE`?